test/functional/helpers.lua

```diff
@@ -26,7 +26,7 @@ local nvim_prog = (
   or global_helpers.test_build_dir .. '/bin/nvim'
 )
 -- Default settings for the test session.
-local nvim_set  = 'set shortmess+=I background=light noswapfile noautoindent'
+local nvim_set  = 'set shortmess+=IS background=light noswapfile noautoindent'
```

Comment:
@justinmk 
Why is `shm+=S` used with tests?  (just confused me.. I think it is better to use default with tests also if possible/feasible).